### ISSUE TITLE
Prelegenci - Ida Tymina niepotrzebny i niedziałający link do Linkedin.

### ISSUE THREAD

**Issue by AgnieszkaS4** [hs3city/define-normalnosc#15]
Opis: W zakładce Prelegenci Ida Tymina ma w swoim profilu ikonkę do Linkedin. Link nie przekierowuje do profilu na Linkedin.
![1  Linkedin Ida Tymina strona](https://user-images.githubusercontent.com/92811043/158237244-5133b764-0fbd-4fe0-8c2c-7fe4c8630be4.PNG)

![1  Linkedin Ida Tymina](https://user-images.githubusercontent.com/92811043/158237340-7e752218-b807-47a5-91fb-02fb5e104c1e.PNG)

W docs była informacja, że Ida nie ma konta na Linkedin. 

Środowisko: Przeglądarka: Google Chrome Wersja 92.0.4515.131
Windows 8.1 6.3.9600.17415

**Comment by maknetaRo** [comment]
Wydaje mi się, że jeśli jesteś zalogowana/y na linkedin to bez problemu znajdziesz ten profil. Jeśli zaś jesteś wylogowana/wylogowany lub nie masz konta to  otrzymasz taki komunikat jak wyżej.
Próbowałam na 2 komputerach (jeden z windows i przeglądarką firefox, drugi z linux i przeglądarką chrome) i telefonie. 
To chyba zależy od ustawień profilu.

**Comment by AgnieszkaS4** [comment]
Tak, po zalogowaniu się na Linkedin profil Idy jest widoczny. Natomiast u innych prelegentów profile na Linkedin są widoczne bez logowania się. Zamykamy w takim razie to zgłoszenie?

**Comment by maknetaRo** [comment]
@tgbdc7 zamykamy to zgłoszenie?
